app/assets/javascripts/modules/collapsible.js

```diff
@@ -5,26 +5,35 @@
   function Collapsible(section){
     this.$section = section;
     this.$clickTarget = this.$section.find('.js-subsection-title');
-    this.$clickTarget.on('click', this.toggle.bind(this));
+    this.$clickTarget.on('click', this.handleClick.bind(this));
     this.addToggle();
 
     this.$section.on('focus', '.js-subsection-body a', this.showSectionWhenLinkFocused.bind(this));
   }
 
+  Collapsible.prototype.handleClick = function handleClick(){
+    this.toggle()
+    this.toggleAttribute()
```

Comment:
Could this be renamed to better convey what it's doing?

```js
this.toggleAriaExpanded()
```

Reply:
done